deflect/config.h

```diff
@@ -40,16 +40,20 @@
 #ifndef DEFLECT_CONFIG_H
 #define DEFLECT_CONFIG_H
 
-#ifdef __cplusplus
-#  ifndef CXX_FINAL_OVERRIDE_SUPPORTED
+#ifdef DEFLECT_USE_CXX03
```

Comment:
Actually, Deflect never built (I guess) or is not meant to built with C++03. At least it doesn't now as there errors in other places (MTQueue, non-static member initialization etc.). Not sure how useful this file actually is then.